common/src/main/java/org/schemarepo/validation/LatestValidator.java

```diff
@@ -0,0 +1,48 @@
+package org.schemarepo.validation;
+
+
+import org.schemarepo.SchemaEntry;
+import org.schemarepo.SchemaValidationException;
+import org.schemarepo.Validator;
+import org.schemarepo.config.Config;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * <p>
+ * A {@link Validator} for validating the provided schema against the
+ * first schema in the iterable in {@link #validate(String, Iterable)}.
+ * </p>
+ * <p>
+ * Uses the {@link ValidationStrategy} provided in the constructor to
+ * validate the schema against the first Schema in the iterable, if it exists,
+ * via {@link ValidationStrategy#validate(String, String)}.
+ * </p>
+ */
+public class LatestValidator implements Validator {

```

Comment:
Another comment in the same vein as my previous one... Perhaps the name can be made more self-explanatory here as well?

```ValidatorAgainstLatestSchema```?

Again, I'm just brainstorming... Not married to any name yet, but I'd like it if the whole thing was as self-explanatory as possible...